Gruntfile.js

```diff
@@ -212,6 +255,38 @@ module.exports = function(grunt) {
 
 	});
 
+	grunt.registerTask( 'kebabToCamelCase', 'Rename any svg attributes to camel case for react', function() {
+		var svgFiles = grunt.file.expand( { filter: 'isFile', cwd: 'svg-min/' }, [ '**/*.svg' ] );
+
+		// Add stuff
+		svgFiles.forEach( function( svgFile ) {
+
+			// Grab the relevant bits from the file contents
+			var fileContent = grunt.file.read( 'svg-min/' + svgFile );
```

Comment:
> Wondering about the svg-min folder. What's the point of it? 
It's just a tmp folder that contains the transformed svgs. It should be fully git-ignored at the moment.

> And should we be camelCasing those props if we ever hope to use them anywhere outside of React?

This was why I moved out the task, and ordered it after the php plugin generation.